core/src/test/java/org/testcontainers/images/ImagePullPolicyTest.java

```diff
@@ -23,10 +20,13 @@
 import java.util.UUID;
 import java.util.concurrent.TimeUnit;
 
+import static org.junit.Assert.fail;
+import static org.mockito.ArgumentMatchers.any;
+
 public class ImagePullPolicyTest {
 
     @ClassRule
-    public static GenericContainer<?> registry = new GenericContainer<>("registry:2")
+    public static GenericContainer<?> registry = new GenericContainer<>(new DockerImageName("registry:2.7.0"))
```

Comment:
I liked the idea of `TestImages`, btw. Perhaps we can/should move such references to that class as well

Reply:
Sure, I missed this one!

Reply:
Updated